src/statmodels.jl

```diff
@@ -174,16 +245,27 @@ end
 const r² = r2
 
 """
-    adjr2(obj::StatisticalModel, variant::Symbol)
-    adjr²(obj::StatisticalModel, variant::Symbol)
+    adjr2(obj::StatisticalModel)
+    adjr²(obj::StatisticalModel)
 
 Adjusted coefficient of determination (adjusted R-squared).
 
 For linear models, the adjusted R² is defined as ``1 - (1 - (1-R^2)(n-1)/(n-p))``, with ``R^2``
 the coefficient of determination, ``n`` the number of observations, and ``p`` the number of
 coefficients (including the intercept). This definition is generally known as the Wherry Formula I.
+"""
+function adjr2(obj::StatisticalModel)
+    n, p = nobs(obj), length(coef(obj)) # Default behavior (p could potentially be incorrect)
```

Comment:
Mmm, indeed it could be incorrect if some coefficients could not be estimated (and would be `NaN`, `0` or anything depending on the implementation). Probably better use `dof` here? Though there's a problem since it includes the dispersion parameter and we have no way to know that here. I see two possible solutions:
- do not define a fallback for now
- add `dof(obj::StatisticalModel, coef_only::Bool = false)` which custom types have to implement, counting only coefficients per se when `coef_only=true`